tgui/packages/tgui/interfaces/Pressurizer.js

```diff
@@ -0,0 +1,220 @@
+                Max
+              </Button>
+            </LabeledList.Item>
+          </LabeledList>
+          <Divider />
+          <LabeledList>
+            <LabeledList.Item label="Fan Status">
+              <Button
+                color={fanState === FanState.Off ? 'bad' : 'default'}
+                onClick={() => handleSetFan(FanState.Off)}
+              >
+                Off
+              </Button>
+              <Button
+                selected={fanState === FanState.In}
+                onClick={() => handleSetFan(FanState.In)}
+              >
+                In
+              </Button>
+              <Button
+                selected={fanState === FanState.Out}
+                onClick={() => handleSetFan(FanState.Out)}
+              >
+                Out
+              </Button>
+            </LabeledList.Item>
+            <LabeledList.Item label="Release Pressure">
+              <Button onClick={() => handleSetPressure(minRelease)}>
+                Min
+              </Button>
+              <NumberInput
+                animated
+                width="7em"
+                value={releasePressure}
+                minValue={minRelease}
+                maxValue={maxRelease}
+                onChange={(e, targetPressure) => handleSetPressure(targetPressure)} />
+              <Button onClick={() => handleSetPressure(maxRelease)}>
+                Max
+              </Button>
+            </LabeledList.Item>
+          </LabeledList>
+        </PortableBasicInfo>
+        <Section
+          title="Material Processing"
+          buttons={(
+            <Button
+              icon="eject"
+              disabled={materialsCount === 0}
+              onClick={() => handleEjectContents()}>
+              Eject
+            </Button>
+          )}>
+          <LabeledList>
+            <LabeledList.Item label="Speed">
+              <Button
+                selected={processRate === 1}
+                onClick={() => handleSetProcessRate(1)}
+              >
+                1
+              </Button>
+              <Button
+                selected={processRate === 2}
+                onClick={() => handleSetProcessRate(2)}
+              >
+                2
+              </Button>
+              <Button
+                selected={processRate === 3}
+                onClick={() => handleSetProcessRate(3)}
+              >
+                3
+              </Button>
+              {!!emagged && (
+                <Button
+                  selected={processRate === 4}
+                  onClick={() => handleSetProcessRate(4)}
+                >
+                  4
+                </Button>
+              )}
+              {!!emagged && (
+                <Button
+                  selected={processRate === 5}
+                  onClick={() => handleSetProcessRate(5)}
+                >
+                  5
+                </Button>
+              )}
```

Comment:
```suggestion
              {!!emagged && (
                <>
                  <Button
                    selected={processRate === 4}
                    onClick={() => handleSetProcessRate(4)}
                  >
                    4
                  </Button>
                  <Button
                    selected={processRate === 5}
                    onClick={() => handleSetProcessRate(5)}
                  >
                    5
                  </Button>
                </>
              )}
```

Reply:
My indents in the suggestion are probably weird as done via copy-paste, but basically: wrap in a fragment and avoid the double `emagged` check.